.ci-scripts/test-for-licenseHeaders.sh

```diff
@@ -59,7 +59,7 @@ getFiles() {
     fi |
         grep -Ev '^./.git' |
         grep -Ev '\.(csv|rdf|ent|dtd|png|gitignore|gitattributes|md|bat|jar|json|couch|view|MF|xz|index)' |
-        grep -Ev '(gradlew|build.gradle|settings.gradle|gradle.properties|gradle/wrapper)' |
+        grep -Ev '(gradlew|build.gradle|settings.gradle|gradle.properties|gradle/wrapper|_config.yml)' |
```

Comment:
I believe this can be deleted, it should be a residue from activating gh pages with some difficulties. I would rather not have this file in the master branch. On our example project no such file was necessary and gh pages was still working.